### PR TITLE
Main nav tweaks

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Reinstate burger menu but for sidebar nav and only pages without children

## QA

- Run ./run and go to http://0.0.0.0:8015/ 
- Check the nav in s/m & l
- There should be no burger on /core pages and the nav should be open by default

### CODE REVIEW

**Comment by grahambancroft** [comment]
[Add burger menu for all but core section](https://trello.com/c/SDuHJLgY/3245-add-burger-menu-for-all-but-core-section)

File: templates/includes/components/sidebar_nav.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<ul class="p-sidebar-nav__list {% if not is_root %}js-mobile-no-collapse{% endif %}">
+<ul id="navigation" class="p-sidebar-nav__list u-clearfix {% if is_root %}js-mobile-no-collapse{% endif %}">
```
Comment (WillMoggridge) [inline templates/includes/components/sidebar_nav.html]: Did you mean to inverse the `if is_root` logic? I thought we just reverted it back to `if not`

File: templates/includes/base/header.html [hunk 2]
```diff
@@ -6,15 +6,18 @@
           developer
         </a>
     </div>
-
-    <nav id="navigation" class="p-navigation__nav u-float--right">
+    {% if 'core' in request.path %}{% else %}
```
Comment (WillMoggridge) [inline templates/includes/base/header.html:9]: Can you create an issue to un-hardcode the core section here after this is merged, please. :)

**Review by WillMoggridge — CHANGES_REQUESTED** [review]
Two points raised, otherwise good

File: templates/includes/components/sidebar_nav.html (hunk 1 above)
Comment (grahambancroft) [reply templates/includes/components/sidebar_nav.html]: Reverted

File: templates/includes/base/header.html (hunk 2 above)
Comment (grahambancroft) [reply templates/includes/base/header.html:9]: sure thing

**Comment by grahambancroft** [comment]
Hi @WillMoggridge this is ready for review again if you’d be so kind

**Review by WillMoggridge — APPROVED** [review]

**Pull request merged**